data/json/items/generic.json

```diff
@@ -2842,6 +2842,21 @@
     "volume": 0,
     "to_hit": -3
   },
+  {
+    "type": "GENERIC",
+    "id": "id_industrial",
+    "looks_like": "id_science",
+    "symbol": ",",
+    "color": "green",
+    "name": "industrial ID card",
+    "description": "This ID card once belonged to a high level techincian.  The reverse side describes protocol for using it; this could grant access at one control panel, if you can find one.",
```

Comment:
```suggestion
    "description": "This ID card once belonged to a high level technician.  The reverse side describes protocol for using it; this could grant access at one control panel, if you can find one.",
```